subcommands/keys/tuf_updates_rotate_online_key.go

```diff
@@ -20,15 +20,15 @@ func init() {
 The new online signing key will be used in both CI and production TUF root.
 
 When you rotate the TUF online signing key:
-- if there are CI or production targets in your factory, they are re-signed using the new key.
-- if there is an active wave in your factory, the TUF online key rotation is not allowed.
-- the new wave cannot be created until you apply the online keys rotation.
+- any CI or production Targets are re-signed using the new key.
+- if there is an active Wave, the TUF online key rotation is not allowed.
+- the new Wave cannot be created until you apply the online keys rotation.
```

Comment:
Capital letters at sentence starts? Or use a semicolon if these are not separate sentences, right?